server/serverConstants.js

```diff
@@ -87,15 +87,15 @@ E.IOS_URL_PREFIX = { envVar: 'IOS_URL_PREFIX', jsonPath: 'IOSUrlPrefix', default
 
 E.DEFAULT_INDEX_PAGE = { envVar: 'DEFAULT_INDEX_PAGE', jsonPath: 'defaultPageIndex', defaultValue: undefined };
 
-E.SIP_ENABLED = { envVar: 'SIP_ENABLED', jsonPath: 'SIP.enabled', defaultValue: false };
+E.SIP_ENABLED = { envVar: 'SIP_ENABLED', jsonPath: 'SIP.enabled', defaultValue: false, parser: parseBool };
```

Comment:
I was going to say, I was 100% sure I put an `if (enabledSip)` before initing the client. Good shout on remembering the parser, embarrassingly I was the one who added the parser in the first place.